bigquery_etl/query_scheduling/templates/airflow_dag.j2

```diff
@@ -95,8 +95,14 @@ with DAG('{{ name }}', default_args=default_args{%+ if schedule_interval != None
             {%+ if task.date_partition_offset and task.date_partition_parameter -%}
             parameters=["{{ task.date_partition_parameter }}:DATE:{% raw %}{{{% endraw %}macros.ds_add(ds, {{ task.date_partition_offset }}){% raw %}}}{%endraw%}"]{% if task.parameters | length > 0 %} + {{ task.parameters }}{% endif %},
             {%+ elif task.parameters | length > 0 -%}
+            {%+ if task.date_partition_parameter -%}
+            parameters={{ task.parameters }} + ["{{ task.date_partition_parameter }}:DATE:{% raw %}{{ds}}{% endraw %}"],
+            {%+ else -%}
             parameters={{ task.parameters }},
             {%+ endif -%}
+            {%+ elif task.date_partition_parameter -%}
+            parameters=["{{ task.date_partition_parameter }}:DATE:{% raw %}{{ds}}{% endraw %}"],
+            {%+ endif -%}
```

Comment:
This is getting pretty complicated haha and I'm spoiled by indentation so it broke my brain for a second lol. Would be nice to move some of this logic to the `query_scheduling` module